### PR TITLE
Reader/Copy Check/Copy Taste Mode

### PR DESCRIPTION
**Pull request by clloyd**
This adds a between print and full site view of the content. 

Certain content is removed (social, header etc)
Certain content is replaced with a placeholder (embeds, videos, tweets)

screen shot 2015-09-10 at 17 16 55

@alastairjardine Info-Bar wording is currently: `Copy taste presents you with a distraction free view of the article content, if you have any feedback please email Digital CMS`

### CODE REVIEW

File: app/com/gu/viewer/views/viewer.scala.html [hunk 1]
```diff
@@ -44,6 +46,8 @@ <h1 class="top-toolbar__page-title">
         @if(previewEnv == "preview") {
             <div data-hidedesktopenabled="true" class="info-bar">To enable desktop preview you need to check and verify mobile first</div>
         }
+        <div data-showreaderactive="true" class="info-bar is-hidden">Copy taste presents you with a distraction free view of the article content, if you have any feedback please email <a href="mailto:digitalcms.dev@@guardian.co.uk">Digital CMS</a></div>
```
Comment (alastairjardine) [inline app/com/gu/viewer/views/viewer.scala.html:49]: Really close. Can you update to:

"Copy taste presents you with a distraction free view of the article content. If you have any feedback please email the Digital CMS team"

File: public/javascript/components/viewer.js [hunk 2]
```diff
@@ -15,30 +16,64 @@ function updateViewer(viewportName, viewportConfig) {
...
+        var styleLink = document.createElement('link');
+        styleLink.href = '/assets/styles/readerMode.css';
+        styleLink.rel = 'stylesheet';
+        styleLink.setAttribute('media', 'screen');
+        styleLink.type = 'text/css';
+
+        viewerEl.contentDocument.body.appendChild(styleLink);
+
+    } catch (e) {
+        console.log('Can\'t enable Reader mode: ', e);
```
Comment (jamespamplin) [inline public/javascript/components/viewer.js:64]: We should also try to provide some feedback to the user if they can't enter reader mode, such as an error bar, or making the button go red. Should be a minor edge case, such as when the user navigates away from a guardian page, so can be done later on.

**Comment by jamespamplin** [comment]
:+1: great work! code looks good. Can you print "Taste mode" too?

**Comment by clloyd** [comment]
@jamespamplin So technically you can. But we've specifically added a "screen" media type to the inserted css, so it won't affect printing.

**Pull request merged**